### PR TITLE
Fix precache readme links and kustomization

### PR DESCRIPTION
**Pull request by vitus133**
Fixes pre-cache samples readme links and kustomization file.
/cc @imiller0 @irinamihai 
Thanks to @yliu127 for finding these

### CODE REVIEW

File: samples/precache/README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
...
 ## Mocks and workarounds ##
-1. Deploy the CRD from [config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml](config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml)
-2. If the CSV object is not present on the hub, it must be mocked. Copy the autogenerated CSV from [bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml](bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml). Change the namespace to `openshift-operators` and adjust the pre-caching image pull spec to your desired location. 
+1. Deploy the CRD from [config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml](../../config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml)
+2. If the CSV object is not present on the hub, it must be mocked. Copy the autogenerated CSV from [bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml](../../bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml). Change the namespace to `openshift-operators` and adjust the pre-caching image pull spec to your desired location.
```
[the remark this answers is not in the document]
Comment (rcarrillocruz) [reply samples/precache/README.md:6]: Just a nit tho, it can go as is just to get this merged.

[the remark this answers is not in the document]
Comment (vitus133) [reply samples/precache/README.md]: Yep, done

**Pull request merged**